### PR TITLE
Add "Mimir / Overview" dashboard

### PR DESCRIPTION
**Pull request by pracucci**
#### What this PR does
As proposed in #3079, in this PR I'm introducing a new dashboard called "Mimir / Overview". The idea is to provide a simple, high level, view over the Mimir cluster. This dashboard is **not** intended to show everything but, on the contrary, to intentionally show few but key metrics to get some insights about the health of a Mimir cluster.

This PR only partially address #3079, because there are few things I intentionally left out (e.g. alerts panel) and that I would like to address in a follow up PR.

Note to reviewers:
- I removed `rulerQueries` from the ruler dashboard. In the past it was used to override ruler dashboard queries because we had some dashboards (e.g. Loki) building on top of it, but that's not the case anymore (I double checked it) and should be safe to remove.
- This PR is expected to be a no-op for any existing dashboard

**How it looks like:**

mimir-overview

#### Which issue(s) this PR fixes or relates to

Part of #3079

#### Checklist

- [ ] Tests updated
- [x] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: operations/mimir-mixin-compiled/dashboards/mimir-reads.json [hunk 1]
```diff
@@ -168,7 +168,7 @@
                   "steppedLine": false,
                   "targets": [
                      {
-                        "expr": "sum(\n  rate(\n    cortex_request_duration_seconds_count{\n      cluster=~\"$cluster\", job=~\"($namespace)/((query-frontend.*|cortex|mimir|mimir-read))\",\n      route=~\"(prometheus|api_prom)_api_v1_query_range\"\n    }[$__rate_interval]\n  )\n)\n",
+                        "expr": "sum(rate(cortex_request_duration_seconds_count{cluster=~\"$cluster\", job=~\"($namespace)/((query-frontend.*|cortex|mimir|mimir-read))\",route=~\"(prometheus|api_prom)_api_v1_query_range\"}[$__rate_interval]))",
```
Comment (pracucci) [inline operations/mimir-mixin-compiled/dashboards/mimir-reads.json:171]: Only removed the newlines.

File: operations/mimir-mixin/dashboards/dashboard-utils.libsonnet [hunk 2]
```diff
@@ -121,6 +121,19 @@ local utils = import 'mixin-utils/utils.libsonnet';
...
     },
 
+  // Returns the URL of a given dashboard, keeping the current time range and variables.
+  dashboardURL(filename)::
+    // Grafana uses a <base> HTML set to the path defined in GF_SERVER_ROOT_URL.
+    // This means that if we create relative links (starting with ".") the browser
+    // will append the base to it, effectively honoring the GF_SERVER_ROOT_URL.
+    //
+    // IMPORTANT: due to an issue with Grafana, this URL works only when opened in a
+    // new browser tab (e.g. link with target="_blank").
```
Comment (pracucci) [inline operations/mimir-mixin/dashboards/dashboard-utils.libsonnet:131]: Note to reviewers: I'm still talking to Grafana team about it. I will either reference an issue or change it, if they will come up with a working alternative.

**Comment by colega** [comment]
The dashboard looks great, my only concern is that we're switching between measuring on the gateway or on the components without really telling the user what we're measuring. Could we at least include `(gateway)` to the panels titles when we're measuring on the Gateway? 

I'd also add a text to the description of the row (the first panel) when it's being measured on the gateway, something like:

> Measured on the gateway, visit the drill down dashboards to see errors and latencies on the internal components.

That would make the debugging easier IMO.

**Review by colega — APPROVED** [review]

**Comment by pracucci** [comment]
> The dashboard looks great, my only concern is that we're switching between measuring on the gateway or on the components without really telling the user what we're measuring. Could we at least include `(gateway)` to the panels titles when we're measuring on the Gateway?
> 
> I'd also add a text to the description of the row (the first panel) when it's being measured on the gateway, something like:
> 
> > Measured on the gateway, visit the drill down dashboards to see errors and latencies on the internal components.
> 
> That would make the debugging easier IMO.

Sounds good idea! I've done it in https://github.com/grafana/mimir/pull/3122/commits/c922d490d980d6fa597f61d9ea8f0f99c74ba28f

Here is a preview of the affected dashboard area, when the gateway is enabled:
Screenshot 2022-10-05 at 12 20 04

**Pull request merged**